backend/node/genvm/std/vector_store.py

```diff
@@ -17,21 +17,24 @@ def __init__(self, model_name: str = None):
         self.vector_data = {}  # Dictionary to store vectors
         self.metadata = {}  # Dictionary to store metadata
         self.model_name = model_name
-        self.next_id = 0
 
-    def add_text(self, text: str, metadata: Any):
+    def add_text(self, text: str, metadata: Any, vector_id: Optional[int] = None):
         """
         Add a new text to the store with its metadata.
 
         Args:
             text (str): The text to be added.
             metadata (Any): The metadata.
+            vector_id (int, optional): The ID for the vector. If not provided, a new ID will be generated.
         """
+        if not vector_id and not isinstance(vector_id, int):
```

Comment:
I think we should fail if we were to get an invalid `vector_id` instead of generating one manually, since it could be confusing for the user
Example: I try to create one with an id of `"123"`, but I didn't realize that I passed a string. So then I see that my vector has id `1`

Reply:
Good point, will adapt

Reply:
changed to only allow integers